Snakefile

```diff
@@ -73,6 +73,18 @@ rule test:
         "py.test --html={output} --self-contained-html"
 
 
+rule build_metadata:
```

Comment:
This could have been its own pull request ;-).

Reply:
Yes, sorry, I might just remove it from here and bring it up later.

Reply:
Nah it's fine. I'm just nagging.